docs/content/concepts/ops-jobs-graphs/jobs.mdx

```diff
@@ -0,0 +1,262 @@
+| <PyObject object="JobDefinition" /> | A job definition. Jobs are the main unit of execution and monitoring in Dagster. Typically constructed using the <PyObject object="job" decorator /> decorator. |
+|                                     |                                                                                                                                                                 |
+
+---
+
+## Creating jobs
+
+Jobs can be created in several ways:
+
+- [Using the <PyObject object="job" decorator /> decorator](#using-the-job-decorator)
+- [From assets](#from-assets)
+- [From a graph](#from-a-graph)
+
+### Using the @job decorator
+
+The simplest way to create a job is to use the <PyObject object="job"/> decorator.
+
+Within the decorated function body, you can use function calls to indicate the dependency structure between the ops/graphs. In this example, the `add_one` op depends on the `return_five` op's output. Because this data dependency exists, the `add_one` op executes after `return_five` runs successfully and emits the required output.
+
+```python file=/concepts/ops_jobs_graphs/simple_job.py
+from dagster import job, op
+
+
+@op
+def return_five():
+    return 5
+
+
+@op
+def add_one(arg):
+    return arg + 1
+
+
+@job
+def do_stuff():
+    add_one(return_five())
+```
+
+When defining a job, you can provide [resources](/concepts/resources), [configuration](/concepts/configuration/config-schema), [hooks](/concepts/ops-jobs-graphs/op-hooks), tags, and an [executor](/deployment/executors) (follow the links for explanation of how to use each of these).
+
+### From assets
+
+Jobs can materialize a fixed set of assets each time they run. Additionally, multiple jobs in the same repository can target overlapping sets of assets:
+
+```python file=/concepts/assets/build_job.py startafter=start_marker endbefore=end_marker
+from dagster import asset, define_asset_job, repository
+
+
+@asset
+def asset1():
+    return [1, 2, 3]
+
+
+@asset
+def asset2(asset1):
+    return asset1 + [4]
+
+
+all_assets_job = define_asset_job(name="all_assets_job")
+asset1_job = define_asset_job(name="asset1_job", selection="asset1")
+
+
+@repository
+def repo():
+    return [asset1, asset2, all_assets_job, asset1_job]
+```
+
+Like regular jobs, jobs that target assets can be placed on [schedules](/concepts/partitions-schedules-sensors/schedules) and [sensors](/concepts/partitions-schedules-sensors/sensors).
+
+### From a graph
+
+Dagster is built around the observation that any data DAG typically contains a logical core of data transformation, which is reusable across a set of environments (e.g. prod, local, staging) or scenarios. The graph usually needs to be "customized" for each environment, by plugging in configuration and services that are specific to that environment.
```

Comment:
Putting this here makes it seem like this only applies to graph-based jobs, but the first line applies to asset-based jobs as well, although in a less direct way.

I would maybe replace it with something more specific, like

> If you want to define the dependencies between ops before binding it to resources, configuration, executors and other features of your environment (prod, local, staging) then ...

I'm struggling a bit with the exact way to word this. What I wrote above is kind of arcane and doesn't exactly roll off the tongue.

Reply:
I took a shot at it, so we'll see.